index.html

```diff
@@ -33,6 +33,24 @@
         </ul>
         <div class="end"></div>
       </nav>
+
+      <!-- Mobile menu  -->
+      <div id="mobile-menu">
+        <div id="close-menu">
+          <a href="#"
+            ><img
+              class="close-icon"
+              src="./assets/icons/close-menu.png"
+              alt="close popup menu"
+          /></a>
+        </div>
+        <ul class="popup-list">
+          <li><a class="menu-list-item" href="#portfolio">Portfolio</a></li>
+          <li><a class="menu-list-item" href="#about">About</a></li>
+          <li><a class="menu-list-item" href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- Kindly ensure that these project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) are met:

- [ ]  When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears, currently when I click or tap the menu options don't go away

- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed, when I click or tap the correct path is not shown I still see the popup it should disappear